client/jetpack-cloud/sections/agency-dashboard/sites-overview/style.scss

```diff
@@ -139,6 +139,43 @@
 	vertical-align: middle;
 	color: var(--studio-gray-5);
 }
+.sites-overview__stats-trend.is-up {
+	vertical-align: middle;
+	display: inline-flex;
+	fill: var(--studio-jetpack-green-40);
+}
+.sites-overview__stats-trend.is-down {
+	vertical-align: middle;
+	display: inline-flex;
+	fill: var(--studio-red-50);
+}
+.sites-overview__stats-trend.same {
+	vertical-align: middle;
+	height: 8px;
+	width: 8px;
+	border-radius: 50%;
+	background: var(--studio-gray-5);
+	display: inline-flex;
+	margin-inline-start: 8px;
+}
+.sites-overview__stats span.shortened-number {
+	vertical-align: middle;
+	color: var(--studio-gray-80);
+	font-size: 0.75rem;
+}
+.sites-overview__stats-trend svg {
+	position: relative;
+	inset-block-start: 0.3rem;
+	inset-inline-start: 0.25rem;
+}
+.site-card__expanded-content-status .sites-overview__stats-trend svg {
```

Comment:
Do we need to append `.site-card__expanded-content-status` here? Looks like the above piece of code is same as this one and we can get rid of one

Reply:
I found that I needed the extra class because the alignment is slight different between desktop and mobile.
  
 I fixed it by using an `@include break-mobile`